data-sources/flutter-android/config.yml

```diff
@@ -0,0 +1,15 @@
+id: flutter-android
+displayName: Flutter Android
+description: |
+  Identify and optimize Flutter Android app performance with New Relic Flutter Android Monitoring.
+icon: logo.png
+install:
+  primary:
+    link:
+      url: https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application/
```

Comment:
@pkudikyala Here replace the button of "Install now" from "See installation docs".